JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -19,21 +19,32 @@ package com.example.jetnews.ui
 import androidx.compose.material.Scaffold
 import androidx.compose.material.rememberScaffoldState
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.SideEffect
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.graphics.Color
 import androidx.navigation.compose.KEY_ROUTE
 import androidx.navigation.compose.currentBackStackEntryAsState
 import androidx.navigation.compose.navigate
 import androidx.navigation.compose.rememberNavController
 import com.example.jetnews.data.AppContainer
 import com.example.jetnews.ui.theme.JetnewsTheme
+import com.google.accompanist.insets.ProvideWindowInsets
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
 import kotlinx.coroutines.launch
 
 @Composable
 fun JetnewsApp(
     appContainer: AppContainer
 ) {
     JetnewsTheme {
+        ProvideWindowInsets {
+            val systemUiController = rememberSystemUiController()
+            SideEffect {
+                systemUiController.setSystemBarsColor(Color.Transparent, darkIcons = false)
+            }
+        }
```

Comment:
All of the app content should be in the `ProvideWindowInsets` (it sets a composition local)

Reply:
Done!